utils/ansible.py

```diff
@@ -0,0 +1,269 @@
+import fauxfactory
+from git import Repo
+import os
+import subprocess
+import yaml
+from shutil import copy, copyfile, rmtree
+from utils import conf
+from utils.providers import get_crud
+from distutils.sysconfig import get_python_lib
+
+
+yml_path = os.path.dirname(__file__) + "/manageiq_ansible_module/"
+library_path = get_python_lib()
+basic_yml_path = os.path.dirname(__file__) + "/ansible_conf/"
+library_path_to_copy_to = basic_yml_path + "library"
+providers_basic_script = "providers_basic_script.yml"
+users_basic_script = "users_basic_script.yml"
+random_token = fauxfactory.gen_alphanumeric(906),
+random_miq_user = fauxfactory.gen_alphanumeric(8)
+local_git_repo = "manageiq_ansible_module"
+pulled_repo_library_path = local_git_repo + "/library"
+remote_git_repo_url = "git://github.com/dkorn/manageiq-ansible-module.git"
```

Comment:
@psav , @mfalesni , @jkrocil do we really want to download module and put somewhere inside our tree ?
Wouldn't it be better to create a PYPI module instead ?

If we are ok with download, what location should be used then ? I suppose utils isn't good place for this module.
btw, there are also other folders which will be created in utils.